config/StoredProcedures/NewProduct.sql

```diff
@@ -0,0 +1,17 @@
+
+use imp_db_dev;
+DROP PROCEDURE IF EXISTS NewProduct;
+
+DELIMITER $$
+CREATE PROCEDURE NewProduct
+(IN _ProductName varchar(255), IN _Description varchar(255),IN _Date date)
+BEGIN
+
+insert into Products values
+(null, _ProductName,_Description,_Date, 1);
```

Comment:
we should use 'default' instead of '1'. It will have the same effect here, but it's more flexible if we decide to change the default value for some reason.